src/bun.js/api/server.zig

```diff
@@ -7216,6 +7258,8 @@ pub fn NewServer(comptime NamespaceType: type, comptime ssl_enabled_: bool, comp
         }
 
         pub fn stop(this: *ThisServer, abrupt: bool) void {
+            this.js_value.deinit();
```

Comment:
I'm not totally sure if this is correct. It can't immediately clear the JS value. You can have more HTTP requests that are about to be sent in the event loop

Reply:
hmmm maybe it is fine actually